modelcard.md

```diff
@@ -22,9 +22,12 @@ model-index:
       type: {task_type}  # Required. Example: automatic-speech-recognition
       name: {task_name}  # Optional. Example: Speech Recognition
     dataset:
-      type: {dataset_type}  # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
-      name: {dataset_name}  # Required. Example: Common Voice zh-CN
-      args: {arg_0}         # Optional. Example: zh-CN
+      type: {dataset_type}         # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
+      name: {dataset_name}         # Required. Example: Common Voice zh-CN
+      config: {dataset_config}     # Optional. Example: zh-CN
+      split: {dataset_split}       # Optional. Example: test
+      revision: {dataset_revision} # Optional. Example: main
+      args: {arg_0}                # Optional. Example: zh-CN
```

Comment:
I see there is some redundancy between the `config` and `args` fields. 

As discussed offline with @lhoestq it makes sense to treat the core parameters of `load_dataset()` at the same level as `type` and bundle all dataset-specific args into `args`